base/errorshow.jl

```diff
@@ -555,6 +560,7 @@ function show_method_candidates(io::IO, ex::MethodError, @nospecialize kwargs=()
                 i += 1
                 print(io, String(take!(line[1])))
             end
+            println(io) # extra newline for spacing to stacktrace
```

Comment:
```suggestion
```

Reply:
Where should the extra newline for an empty line go, if not here? The stacktrace printing happens in a completely different part of the stack, it doesn't even know about being in a thrown exception, I don't think.

Reply:
I thought that this was relating to the same thing as my previous comment. I'll click resolve